documentation/doc_sncf/source/integration.rst

```diff
@@ -1,29 +1,60 @@
-Navitia documentation: v1 interface 
+SNCF API documentation
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. contents:: Index
 
 Overview
 ========
 
-This document describes how to call navitia via the v1 interface, and the returned resources.
+This document describes how to use the SNCF API based on navitia via the v1 interface, and the returned resources.
+Navitia is an Open Source software developed by Canal TP. (www.canaltp.fr)
+
+The SNCF API handle to :
+
+* compute journey from and to "station" or "administrative region"
+* display next departures or arrivals
+* display route schedules
+* lexicographical search (autocomplete)
+
+The SNCF API contains theorical train data for the following commercial modes : TGV, TER, Transilien, Intercités.
+
+Read our lexicon : https://github.com/OpenTransport/vocabulary/blob/master/vocabulary.md
+
+Authentication
+================
+
+You must authenticate to use **SNCF API**. When you register we give you an authentication key to the API.
+
+There is two ways for authentication:
```

Comment:
"There is two ways for the authenticate" no ? (not sure at all there, but seems strange to me)